packages/l10n/scripts/missing-countries.js

```diff
@@ -0,0 +1,82 @@
+const missingCountries = {
```

Comment:
Let's add some comments here to explain a bit the context.

For instance, that these are legacy countries and do not exist anymore.